qiskit_experiments/library/calibration/rough_amplitude_cal.py

```diff
@@ -200,6 +201,8 @@ class RoughXSXAmplitudeCal(RoughAmplitudeCal):
         qiskit_experiments.library.characterization.rabi.Rabi
     """
 
+    __outcome__ = "rabi_rate"
```

Comment:
Why do these need to be class variables?

Reply:
Otherwise you need to write `__init__` for every subclass experiments just to override parameter name in the analysis option. Previously you define `_default_analysis_options_` in the experiment but this is definitely not the way to go (because we decouple the experiment and analysis).